games-fps/sauerbraten/sauerbraten-2020.12.29.ebuild

```diff
@@ -36,13 +36,15 @@ BDEPEND="virtual/pkgconfig"
 
 PATCHES=(
 	# Respect CXXFLAGS, LDFLAGS
-	"${FILESDIR}"/${PN}-2020.12.27-respect-FLAGS-don-t-strip-symbols.patch
+	"${FILESDIR}"/${PN}-2020.12.29-respect-FLAGS-don-t-strip-symbols.patch
```

Comment:
It is common practice to keep the version in the patch's file name, so it eventually becomes the version of the package that introduced the patch.

Reply:
Alright, I will fix this.